tools/AndroidDependencies/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.2.8f1
+m_EditorVersion: 2018.2.14f1
```

Comment:
Having Unity version sprinkled over various repositories is sad :(

Reply:
Keep bugging Unity about this weird workflow